install-task-doc.sh

```diff
@@ -0,0 +1,18 @@
+# Copyright 2016, EMC, Inc.
+
+# Generate task doc from task schemas
+cd ./node_modules/on-tasks/
```

Comment:
how about if users doesn't run the `npm install` before running this script? I suggest either check whether the `./node_modules/on-tasks/` exists or add `set -e` at the beginning of this script.